### PR TITLE
Refactor current query terms

### PR DESCRIPTION
**Pull request by VasilyShelkov**
-Changed #searchKeyword css to #addSearchTerm to be consistent with name of component
-Extracted AddSearchTerm into its own component
-Using ids with jquery instead of react refs since it was the way to achieve the onclick and focus in the least amount of code and also so selecting the container and addsearchterm is consistently using ids instead of one using refs and the other using id selector 

@Winwardo ready to be checked and merged if you're happy with it.

### CODE REVIEW

File: src/client/search/AddSearchTerm.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  const keywordSearchStyle = {
+    display:'none',
+    paddingLeft:20,
+    paddingTop:10,
+    marginTop:10,
+    borderTop:1,
+    borderTop: '2px dashed #D3D5D8',
+  };
+  return (
+    <div className="ui fluid big transparent input"
```
Comment (Winwardo) [inline src/client/search/AddSearchTerm.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2226118788/issues/source?bid=3148782&fileBranchId=3174638#l15)

File: src/client/search/AddSearchTerm.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      style={keywordSearchStyle}
+      id="searchTermContainer">
```
Comment (Winwardo) [inline src/client/search/AddSearchTerm.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2226118788/issues/source?bid=3148782&fileBranchId=3174638#l17)

File: src/client/search/CurrentQueryTerms.js [hunk 3]
```diff
@@ -1,56 +1,30 @@
...
 
 let nextSearchTermId = 0;
-let CurrentQueryTerms = ({ dispatch }) => {
-  let showSearchkeyword = false;
-  let searchKeyword;
+
+const CurrentQueryTerms = () => {
+  const showAddSearchTerm = () => {
+    //searchTermContainer and addSearchTerm are in ./AddSearchTerm
+    if ($('#searchTermContainer').is(':hidden')) {
```
Comment (Winwardo) [inline src/client/search/CurrentQueryTerms.js:9]: Please add to the comment that you're aware this is a bad practise, but is done for the sake for simplicity and ease of coding

File: src/client/search/CurrentQueryTerms.js [hunk 4]
```diff
@@ -1,56 +1,30 @@
...
-
-                 onKeyDown={(e) => {
-              if (e.keyCode == 13) {
-                dispatch(addSearchTerm(nextSearchTermId++, searchKeyword.value));
-                searchKeyword.value = '';
-              }
-						}}/>
-          <i className="link remove circle icon"></i>
-        </div>
+        <AddSearchTerm addSerchTermRef={node => searchTermRef = node}/>
```
Comment (Winwardo) [inline src/client/search/CurrentQueryTerms.js]: *addSearchTermRef, not addSerchTermRef

**Comment by Winwardo** [comment]
All seems good :+1:

**Comment by VasilyShelkov** [comment]
@Winwardo Ready to push now that fixed your comments :)

**Pull request merged**